### ISSUE TITLE
2024-12-11, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#17]
# Today's Arxiv Papers

(1) [BiMediX2: Bio-Medical EXpert LMM for Diverse Medical Modalities](http://arxiv.org/abs/2412.07769v1), 2024-12-10

(2) [Bayesian Optimization of Antibodies Informed by a Generative Model of Evolving Sequences](http://arxiv.org/abs/2412.07763v1), 2024-12-10

(3) [Zero-Shot ATC Coding with Large Language Models for Clinical Assessments](http://arxiv.org/abs/2412.07743v1), 2024-12-10

(4) [Granite Guardian](http://arxiv.org/abs/2412.07724v1), 2024-12-10

(5) [DriveMM: All-in-One Large Multimodal Model for Autonomous Driving](http://arxiv.org/abs/2412.07689v1), 2024-12-10

(6) [Privacy-Preserving Customer Support: A Framework for Secure and Scalable Interactions](http://arxiv.org/abs/2412.07687v1), 2024-12-10

(7) [TRIM: Token Reduction and Inference Modeling for Cost-Effective Language Generation](http://arxiv.org/abs/2412.07682v1), 2024-12-10

(8) [RAZOR: Sharpening Knowledge by Cutting Bias with Unsupervised Text Rewriting](http://arxiv.org/abs/2412.07675v1), 2024-12-10

(9) [Ask Humans or AI? Exploring Their Roles in Visualization Troubleshooting](http://arxiv.org/abs/2412.07673v1), 2024-12-10

(10) [FlexLLM: Exploring LLM Customization for Moving Target Defense on Black-Box LLMs Against Jailbreak Attacks](http://arxiv.org/abs/2412.07672v1), 2024-12-10

(11) [Automating Business Intelligence Requirements with Generative AI and Semantic Search](http://arxiv.org/abs/2412.07668v1), 2024-12-10

(12) [Searching for Structure: Investigating Emergent Communication with Large Language Models](http://arxiv.org/abs/2412.07646v1), 2024-12-10

(13) [TrojanWhisper: Evaluating Pre-trained LLMs to Detect and Localize Hardware Trojans](http://arxiv.org/abs/2412.07636v1), 2024-12-10

(14) [ChocoLlama: Lessons Learned From Teaching Llamas Dutch](http://arxiv.org/abs/2412.07633v1), 2024-12-10

(15) [OmniDocBench: Benchmarking Diverse PDF Document Parsing with Comprehensive Annotations](http://arxiv.org/abs/2412.07626v1), 2024-12-10

(16) [DRUM: Learning Demonstration Retriever for Large MUlti-modal Models](http://arxiv.org/abs/2412.07619v1), 2024-12-10

(17) [Adapting to Non-Stationary Environments: Multi-Armed Bandit Enhanced Retrieval-Augmented Generation on Knowledge Graphs](http://arxiv.org/abs/2412.07618v1), 2024-12-10

(18) [DiffSensei: Bridging Multi-Modal LLMs and Diffusion Models for Customized Manga Generation](http://arxiv.org/abs/2412.07589v1), 2024-12-10

(19) [Scaling Sequential Recommendation Models with Transformers](http://dx.doi.org/10.1145/3626772.3657816), 2024-12-10

(20) [Automatic Database Configuration Debugging using Retrieval-Augmented Language Models](http://arxiv.org/abs/2412.07548v1), 2024-12-10

(21) [CoPrUS: Consistency Preserving Utterance Synthesis towards more realistic benchmark dialogues](http://arxiv.org/abs/2412.07515v1), 2024-12-10

(22) [Ontology-driven Prompt Tuning for LLM-based Task and Motion Planning](http://arxiv.org/abs/2412.07493v1), 2024-12-10

(23) [Dynamic Ensemble Reasoning for LLM Experts](http://arxiv.org/abs/2412.07448v1), 2024-12-10

(24) [Optimizing Alignment with Less: Leveraging Data Augmentation for Personalized Evaluation](http://arxiv.org/abs/2412.07429v1), 2024-12-10

(25) [Beyond Search Engines: Can Large Language Models Improve Curriculum Development?](http://dx.doi.org/10.1007/978-3-031-72312-4_17), 2024-12-10

(26) [Generating Knowledge Graphs from Large Language Models: A Comparative Study of GPT-4, LLaMA 2, and BERT](http://arxiv.org/abs/2412.07412v1), 2024-12-10

(27) [MoDULA: Mixture of Domain-Specific and Universal LoRA for Multi-Task Learning](http://arxiv.org/abs/2412.07405v1), 2024-12-10

(28) [CMT: A Memory Compression Method for Continual Knowledge Learning of Large Language Models](http://arxiv.org/abs/2412.07393v1), 2024-12-10

(29) [Algorithmic Phase Transitions in Language Models: A Mechanistic Case Study of Arithmetic](http://arxiv.org/abs/2412.07386v1), 2024-12-10

(30) [SpecFuse: Ensembling Large Language Models via Next-Segment Prediction](http://arxiv.org/abs/2412.07380v1), 2024-12-10

(31) [My Words Imply Your Opinion: Reader Agent-Based Propagation Enhancement for Personalized Implicit Emotion Analysis](http://arxiv.org/abs/2412.07367v1), 2024-12-10

(32) [Towards Predictive Communication with Brain-Computer Interfaces integrating Large Language Models](http://arxiv.org/abs/2412.07355v1), 2024-12-10

(33) [Frame Representation Hypothesis: Multi-Token LLM Interpretability and Concept-Guided Text Generation](http://arxiv.org/abs/2412.07334v1), 2024-12-10

(34) [ConceptSearch: Towards Efficient Program Search Using LLMs for Abstraction and Reasoning Corpus (ARC)](http://arxiv.org/abs/2412.07322v1), 2024-12-10

(35) [The Rise and Down of Babel Tower: Investigating the Evolution Process of Multilingual Code Large Language Model](http://arxiv.org/abs/2412.07298v1), 2024-12-10

(36) [Enhancing Relation Extraction via Supervised Rationale Verification and Feedback](http://arxiv.org/abs/2412.07289v1), 2024-12-10

(37) [HARP: Hesitation-Aware Reframing in Transformer Inference Pass](http://arxiv.org/abs/2412.07282v1), 2024-12-10

(38) [MemHunter: Automated and Verifiable Memorization Detection at Dataset-scale in LLMs](http://arxiv.org/abs/2412.07261v1), 2024-12-10

(39) [Label-Confidence-Aware Uncertainty Estimation in Natural Language Generation](http://arxiv.org/abs/2412.07255v1), 2024-12-10

(40) [KULTURE Bench: A Benchmark for Assessing Language Model in Korean Cultural Context](http://arxiv.org/abs/2412.07251v1), 2024-12-10

(41) [Filling Memory Gaps: Enhancing Continual Semantic Parsing via SQL Syntax Variance-Guided LLMs without Real Data Replay](http://arxiv.org/abs/2412.07246v1), 2024-12-10

(42) [CBraMod: A Criss-Cross Brain Foundation Model for EEG Decoding](http://arxiv.org/abs/2412.07236v1), 2024-12-10

(43) [Towards Automated Cross-domain Exploratory Data Analysis through Large Language Models](http://arxiv.org/abs/2412.07214v1), 2024-12-10

(44) [IntellectSeeker: A Personalized Literature Management System with the Probabilistic Model and Large Language Model](http://arxiv.org/abs/2412.07213v1), 2024-12-10

(45) [EDiT: A Local-SGD-Based Efficient Distributed Training Method for Large Language Models](http://arxiv.org/abs/2412.07210v1), 2024-12-10

(46) [MAPLE: A Framework for Active Preference Learning Guided by Large Language Models](http://arxiv.org/abs/2412.07207v1), 2024-12-10

(47) [PrisonBreak: Jailbreaking Large Language Models with Fewer Than Twenty-Five Targeted Bit-flips](http://arxiv.org/abs/2412.07192v1), 2024-12-10

(48) [When Graph Meets Retrieval Augmented Generation for Wireless Networks: A Tutorial and Case Study](http://arxiv.org/abs/2412.07189v1), 2024-12-10

(49) [Exploring What Why and How: A Multifaceted Benchmark for Causation Understanding of Video Anomaly](http://arxiv.org/abs/2412.07183v1), 2024-12-10

(50) [Post-Training Statistical Calibration for Higher Activation Sparsity](http://arxiv.org/abs/2412.07174v1), 2024-12-10

(51) [Breaking the Stage Barrier: A Novel Single-Stage Approach to Long Context Extension for Large Language Models](http://arxiv.org/abs/2412.07171v1), 2024-12-10

(52) [Political Actor Agent: Simulating Legislative System for Roll Call Votes Prediction with Large Language Models](http://arxiv.org/abs/2412.07144v1), 2024-12-10

(53) [A Review of Human Emotion Synthesis Based on Generative Technology](http://arxiv.org/abs/2412.07116v1), 2024-12-10

(54) [Exploring Coding Spot: Understanding Parametric Contributions to LLM Coding Performance](http://arxiv.org/abs/2412.07113v1), 2024-12-10

(55) [Predictable Emergent Abilities of LLMs: Proxy Tasks Are All You Need](http://arxiv.org/abs/2412.07111v1), 2024-12-10

(56) [On Evaluating the Durability of Safeguards for Open-Weight LLMs](http://arxiv.org/abs/2412.07097v1), 2024-12-10

(57) [Defensive Dual Masking for Robust Adversarial Defense](http://arxiv.org/abs/2412.07078v1), 2024-12-10

(58) [MoE-CAP: Cost-Accuracy-Performance Benchmarking for Mixture-of-Experts Systems](http://arxiv.org/abs/2412.07067v1), 2024-12-10

(59) [Large Language Models: An Applied Econometric Framework](http://arxiv.org/abs/2412.07031v1), 2024-12-09

(60) [Sequential Compression Layers for Efficient Federated Learning in Foundational Models](http://arxiv.org/abs/2412.07021v1), 2024-12-09

(61) [Assessing the Impact of Conspiracy Theories Using Large Language Models](http://arxiv.org/abs/2412.07019v1), 2024-12-09

(62) [Asynchronous LLM Function Calling](http://arxiv.org/abs/2412.07017v1), 2024-12-09

(63) [ProVision: Programmatically Scaling Vision-centric Instruction Data for Multimodal Language Models](http://arxiv.org/abs/2412.07012v1), 2024-12-09

(64) [Phaedrus: Exploring Dynamic Application Behavior with Lightweight Generative Models and Large-Language Models](http://arxiv.org/abs/2412.06994v1), 2024-12-09

(65) [AutoReason: Automatic Few-Shot Reasoning Decomposition](http://arxiv.org/abs/2412.06975v1), 2024-12-09

(66) [Effective Text Adaptation for LLM-based ASR through Soft Prompt Fine-Tuning](http://arxiv.org/abs/2412.06967v1), 2024-12-09

(67) [Bridging Conversational and Collaborative Signals for Conversational Recommendation](http://arxiv.org/abs/2412.06949v1), 2024-12-09

(68) [PyraNet: A Multi-Layered Hierarchical Dataset for Verilog](http://arxiv.org/abs/2412.06947v1), 2024-12-09

(69) [Creating a Cooperative AI Policymaking Platform through Open Source Collaboration](http://arxiv.org/abs/2412.06936v1), 2024-12-09

(70) [Non-Prehensile Tool-Object Manipulation by Integrating LLM-Based Planning and Manoeuvrability-Driven Controls](http://arxiv.org/abs/2412.06931v1), 2024-12-09

(71) [Using Large Language Models to Assign Partial Credits to Students' Problem-Solving Process: Grade at Human Level Accuracy with Grading Confidence Index and Personalized Student-facing Feedback](http://arxiv.org/abs/2412.06910v1), 2024-12-09

(72) [SafeWatch: An Efficient Safety-Policy Following Video Guardrail Model with Transparent Explanations](http://arxiv.org/abs/2412.06878v1), 2024-12-09

(73) [Training Large Language Models to Reason in a Continuous Latent Space](http://arxiv.org/abs/2412.06769v1), 2024-12-09

(74) [Why Do Developers Engage with ChatGPT in Issue-Tracker? Investigating Usage and Reliance on ChatGPT-Generated Code](http://arxiv.org/abs/2412.06757v1), 2024-12-09

(75) [LLMs for Generalizable Language-Conditioned Policy Learning under Minimal Data Requirements](http://arxiv.org/abs/2412.06877v1), 2024-12-09

(76) [Refusal Tokens: A Simple Way to Calibrate Refusals in Large Language Models](http://arxiv.org/abs/2412.06748v1), 2024-12-09

(77) [ONEBench to Test Them All: Sample-Level Benchmarking Over Open-Ended Capabilities](http://arxiv.org/abs/2412.06745v1), 2024-12-09

(78) [JAPAGEN: Efficient Few/Zero-shot Learning via Japanese Training Dataset Generation with LLM](http://arxiv.org/abs/2412.06738v1), 2024-12-09

(79) [AutoDCWorkflow: LLM-based Data Cleaning Workflow Auto-Generation and Benchmark](http://arxiv.org/abs/2412.06724v1), 2024-12-09

(80) [OmniEvalKit: A Modular, Lightweight Toolbox for Evaluating Large Language Model and its Omni-Extensions](http://arxiv.org/abs/2412.06693v1), 2024-12-09

(81) [Exploring Critical Testing Scenarios for Decision-Making Policies: An LLM Approach](http://arxiv.org/abs/2412.06684v1), 2024-12-09

(82) [Toward LLM-Agent-Based Modeling of Transportation Systems: A Conceptual Framework](http://arxiv.org/abs/2412.06681v1), 2024-12-09

(83) [I Don't Know: Explicit Modeling of Uncertainty with an [IDK] Token](http://arxiv.org/abs/2412.06676v1), 2024-12-09

(84) [ILLUME: Illuminating Your LLMs to See, Draw, and Self-Enhance](http://arxiv.org/abs/2412.06673v1), 2024-12-09

(85) [MuMu-LLaMA: Multi-modal Music Understanding and Generation via Large Language Models](http://arxiv.org/abs/2412.06660v1), 2024-12-09

(86) [GEAR: A Simple GENERATE, EMBED, AVERAGE AND RANK Approach for Unsupervised Reverse Dictionary](http://arxiv.org/abs/2412.06654v1), 2024-12-09

(87) [Chatbots im Schulunterricht: Wir testen das Fobizz-Tool zur automatischen Bewertung von Hausaufgaben](http://arxiv.org/abs/2412.06651v2), 2024-12-09

(88) [MAVias: Mitigate any Visual Bias](http://arxiv.org/abs/2412.06632v1), 2024-12-09

(89) [AI TrackMate: Finally, Someone Who Will Give Your Music More Than Just "Sounds Great!"](http://arxiv.org/abs/2412.06617v1), 2024-12-09

(90) [3D Spatial Understanding in MLLMs: Disambiguation and Evaluation](http://arxiv.org/abs/2412.06613v1), 2024-12-09

(91) [Examining the Use and Impact of an AI Code Assistant on Developer Productivity and Experience in the Enterprise](http://arxiv.org/abs/2412.06603v1), 2024-12-09

(92) [Towards Controllable Speech Synthesis in the Era of Large Language Models: A Survey](http://arxiv.org/abs/2412.06602v1), 2024-12-09

(93) [Anchoring Bias in Large Language Models: An Experimental Study](http://arxiv.org/abs/2412.06593v1), 2024-12-09

(94) [Data Quality Enhancement on the Basis of Diversity with Large Language Models for Text Classification: Uncovered, Difficult, and Noisy](http://arxiv.org/abs/2412.06575v2), 2024-12-09

(95) [Applications and Implications of Large Language Models in Qualitative Analysis: A New Frontier for Empirical Software Engineering](http://arxiv.org/abs/2412.06564v2), 2024-12-09

(96) [Challenges and Opportunities for Visual Analytics in Jurisprudence](http://arxiv.org/abs/2412.06543v1), 2024-12-09

(97) [Sloth: scaling laws for LLM skills to predict multi-benchmark performance across families](http://arxiv.org/abs/2412.06540v1), 2024-12-09

(98) [Understanding Factual Recall in Transformers via Associative Memories](http://arxiv.org/abs/2412.06538v1), 2024-12-09

(99) [The Fusion of Large Language Models and Formal Methods for Trustworthy AI Agents: A Roadmap](http://arxiv.org/abs/2412.06512v1), 2024-12-09

(100) [Small Languages, Big Models: A Study of Continual Training on Languages of Norway](http://arxiv.org/abs/2412.06484v1), 2024-12-09
